tests/html_tags/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2552,6 +2569,22 @@ printWidth: 1
   />
 </span>
 
+<!-- self-closing -->
+<span
+  ><input
+    type="checkbox"
+  />
+</span>
+<span
+  ><span
+    ><input
+      type="checkbox" /></span
```

Comment:
Seems all good except this one.

Reply:
Seems breakable [playground](https://deploy-preview-7395--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAeAzgBwIZQHwY76oCWUmArjAAQwCemcAvADrgAWcYA1gEYQAPNrQbM2YTj35CQIxqw5c+g4fXnjJymXLGKpK2Wt0Sl01aIUn92o5c1nDFjaYM67Lm071bz67w4B6AgCsXGDQ-BAAGhAITBgSaHRkUGwAJzSIAHcABXSEZJRsADcIEgATaJBeNOweOBgAZRwwMgBzZBg0ijgY9hgAWwAbAHV2EnhQsDhGgomSYom6ZHB0ZJiydDg0mBzatoHsZAAzbCGtmIArdAEAIVr6puwBuAAZMjgTs4uQa4FG9pDOAARQoEHgX3OvRAODSWzSK36wyqmDSZBgIwqMHYyAAHAAGGKoiBbEa1TArVFweHFT4xACOYPgezihRA2HQAFooHA4OU+VU0nBGSQhXtsAcjkhTlCYlsBiROt1oehASCmZ9pd9oTBsLxMeVscgAEwxLrYEhDdoAYQgA0OKyg0DpIAoWwAKnrCjKfsUegBJKD82CNMBo+IAQSDjXoQMhWwAvgmgA), should be good

Reply:
I think we should not break here, but it is the other problem and we can open issue about it and solve in an other PR

Reply:
no, this need break. Its `printWidth=1` test

Reply:
Oh yes :+1: